src/components/molecules/Button/styles.module.scss

```diff
@@ -0,0 +1,117 @@
+@import 'src/styles/variables';
+@import 'src/styles/mixins';
+
+.baseButton {
+  display: flex;
+}
+
+.buttonText {
+  order: 2;
+  @include buttonTextStyle;
+  line-height: $line-height-24;
+}
+
+.btn {
+  $self: &;
+  appearance: none;
+  display: inline-flex;
+  align-items: center;
+  background: none;
+  border: 0;
+  color: $color-white;
+  cursor: pointer;
+  overflow: visible;
+  padding: 8px 40px;
+  text-decoration: none;
+  border-radius: $border-radius-20;
+  white-space: nowrap;
+  background-color: $color-primary;
+
+  &:focus {
+    outline: none;
+  }
+
+  &--disabled {
```

Comment:
BEM styling, was mentioned above. For disabled we actually don't need a new class name at all. Disabled is also passed to the wrapping "BaseButton" and can be targeted with the ":disabled" pseudo-selector.